Assets/MixedRealityToolkit.SDK/Profiles/MixedRealityControllerConfigurationProfile.md

```diff
@@ -93,47 +100,53 @@ The Action each input can perform, is completely up to you.
 > See the [Input Action]() configuration profile for more information.
 
 ## Example Models
----
 
 The models provided in the Mixed Reality Toolkit "Standard Assets" folder are as follows:
 
 > These can be found in "Mixed Reality Toolkit SDK / Standard Assets / Controllers"
 
 ### Debug Controllers
+
 The Mixed Reality Toolkit provides a set of basic Gizmo style controllers, used to help align your models to the controller position output by the SDK, to help with offset settings.
 > Note, the ability to display BOTH the Gizmo and the controller models isn't supported as yet. This will be included in a future release.
 
 We recommend applying the following Prefab transform values to align the controller model when using to align with the users view:
+
 * Position - no change
 * Rotation - X 90 (left) -90 (right), Y 0 , Z 0
 
 ## Additional models available to download
----
 
-### Oculus Hands (provided via the Oculus Opensource project)
-Oculus provide two basic hand models, including animation (although not available through the visualizer currently).  
+### Oculus Hands (provided via the Oculus open source project)
```

Comment:
```suggestion
### Oculus hands (provided via the Oculus open source project)
```

Reply:
I have a follow-up PR I think you're going to like 😉